docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -1315,6 +1315,12 @@ \subsubsection{Certificate Precedence and Validity}
 \item
   An operational key certificate with a higher counter overrides one with a
   lower counter.
+\item
+  Also, we require that within any given chain, if there are two blocks A and B
+  signed using operational key certificates issued by the same cold key, if A is
+  an older block than B, the counter in the operational certificate in the
+  witness of B must be at least as large as the one in the counter in the
```

Comment:
I think "in the witness of" B isn't right, it's just in the cert itself. The cert is in the header, and then the header is signed. I think it's enough to say in the cert, or in the cert in the header of B.

Reply:
Right. I'll also change it to "certificate in the header of A" in the next line.